sea.js

```diff
@@ -825,16 +825,21 @@
       return gun;
     }
     // now that we have created a user, we want to authenticate them!
-    User.prototype.auth = function(alias, pass, cb, opt){
+    User.prototype.auth = function(){
+      const alias = typeof arguments[0] === 'string' ? arguments[0] : null
```

Comment:
i'd keep arguments as part of the function signature, otherwise its not very readable or dev friendly

Reply:
yes, let's refactor it.